markdown/ch7.markdown

```diff
@@ -95,11 +96,32 @@ whether jiggling it in one direction or another reduces their
 dissimilarity.  By doing this repeatedly, we can try to find the
 correct alignment.
 
+You'll remember our astronaut - Eileen Collins - from chapter 3.
+We will be shifting this image by 50 pixels to the right then comparing it back
+to the original until we
+find the shift that best matches. Obviously this is a silly thing to do, as we
+know the original orientation, but this way we know the truth. Here's the
+original and shifted image.
+As in previous chapters, images will just be NumPy arrays.
```

Comment:
Will remove. I was just keeping this in since I assumed you wanted to make this point for some reason.